app/vmstorage/servers/vminsert.go

```diff
@@ -134,7 +136,7 @@ var (
 )
 
 // MustStop gracefully stops s so it no longer touches s.storage after returning.
-func (s *VMInsertServer) MustStop() {
+func (s *VMInsertServer) MustStop(grace time.Duration) {
```

Comment:
nit: would be nice to log that there is a grace shutdown and that vmstorage is going to close connections one-by-one on the time interval